.misc/travis_build_and_test.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-py.test
+set -e
```

Comment:
maybe `set -e` should go into a separate commit? Or maybe into the previous one? Up to you :+1: